habitat_baselines/rl/ddppo/policy/running_mean_and_var.py

```diff
@@ -4,18 +4,34 @@
 # This source code is licensed under the MIT license found in the
 # LICENSE file in the root directory of this source tree.
 
+
 import torch
 from torch import Tensor
 from torch import distributed as distrib
 from torch import nn as nn
 
 
+def welford_update(mean, var, count, new_mean, new_var, new_count):
+    m_a = var * count
+    m_b = new_var * new_count
+    M2 = (
+        m_a
+        + m_b
+        + (new_mean - mean).pow(2) * count * new_count / (count + new_count)
+    )
+
+    var = M2 / (count + new_count)
```

Comment:
Based on [Parallel algorithm ](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance#Parallel_algorithm) it should be:
```suggestion
    var = M2 / (count + new_count - 1)
```